fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -0,0 +1,43 @@
+
+    # Prevent adding duplicate handlers if this function is called multiple times
+    if logger.hasHandlers():
+        return logger
+
+    # Create a console handler
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(level)
+
+    # Create a formatter and set it for the handler
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)